source/server/listener_manager_impl.cc

```diff
@@ -671,10 +671,13 @@ void ListenerImpl::setSocket(const Network::SocketSharedPtr& socket) {
 
 ListenerManagerImpl::ListenerManagerImpl(Instance& server,
                                          ListenerComponentFactory& listener_factory,
-                                         WorkerFactory& worker_factory)
-    : server_(server), factory_(listener_factory), stats_(generateStats(server.stats())),
+                                         WorkerFactory& worker_factory,
+                                         bool enable_dispatcher_stats)
```

Comment:
Note: I put this here, rather than as a param in `startWorkers()` or a new `initializeStats()` method, to avoid touching the `ListenerManager` interface. I could do this differently if anybody feels I should...